bootstrapvz/common/fs/logicalvolume.py

```diff
@@ -0,0 +1,30 @@
+from bootstrapvz.base.fs.volume import Volume
+from bootstrapvz.common.tools import log_check_call
+
+
+class LogicalVolume(Volume):
+
+    def create(self, image_path):
+        self.image_path = image_path
+        self.fsm.create(image_path=image_path)
+
+    def _before_create(self, e):
+        lv_size = str(self.size.bytes.get_qty_in('MiB'))
+        vg, lv = e.image_path.split('/')[2:4]
```

Comment:
The image_path is created in [https://github.com/Exy13/bootstrap-vz/pull/3/commits/78cc0aa1d21a7dc20eb290ffd968fa64f4c87b67#diff-ef5430551d8dc6ff04216608239fd813](url). Therefore it always has the same format as it is created from the manifest.